### PR TITLE
Add fast integration job for central Application Gateway

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Add optional fast integration jobs for central Application Gateway for open-source and Compass scenarios. Jobs are optional as this is a new feature and we don't want to break pipelines for potential critical PRs. If they are stable enough (and we hope they are) they could be changed to required. There are two separate jobs as the new Gateway could work in two modes - open-source and Compass one
- Jobs run fast integration pipelines prepared in [this PR](https://github.com/kyma-project/kyma/pull/11152) 

See [the issue](https://github.com/kyma-project/kyma/issues/10361), [the first](https://github.com/kyma-project/kyma/issues/10361#issuecomment-825549125) and [the second](https://github.com/kyma-project/kyma/issues/10361#issuecomment-851853012) comments

### CODE REVIEW

File: vpath/pjtester.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pjNames:
+  - pjName: "pre-main-kyma-integration-k3s-central-app-gateway-compass"
```
Comment (rafalpotempa) [inline vpath/pjtester.yaml:2]: Should it be eventually removed?

Comment (franpog859) [reply vpath/pjtester.yaml:2]: Yes, It's here just for the testing purposes

**Review by adamwalach — APPROVED** [review]

**Pull request merged**